### ISSUE TITLE
IE not working with reactor

### ISSUE THREAD

**Issue by sparsh00** [SeleniumHQ/selenium#8563]
## 🐛 Bug Report

Unable to run complete test on IE while running jar with `-Dwebdriver.http.factory=reactor` in case of 4.0.0-alpha-6, working fine with `okhttp`. Failing to open url.

OS:  Windows 10
Browser: Internet Explorer
Browser version: 11
Browser Driver version: 3.141.5.0 
Selenium Grid version (if applicable): 4.0.0-aplha-6

**Comment by Dharin-shah** [comment]
Error from driver logs 
```
`D 2020-08-12 07:46:08:571 c:\projects\webdriver\cpp\webdriver-server\server.cc(338) Command: POST /session/af254a14-cbe4-4aea-91ad-56b6f4f1f628/url
D 2020-08-12 07:46:08:571 c:\projects\webdriver\cpp\webdriver-server\command.cc(36) Raw JSON command: { "name" : "get", "locator" : { "sessionid" : "af254a14-cbe4-4aea-91ad-56b6f4f1f628" }, "parameters" :  }
W 2020-08-12 07:46:08:571 c:\projects\webdriver\cpp\webdriver-server\command.cc(49) Failed to parse configuration due to * Line 1, Column 105
  Syntax error: value, object or array expected.
```
`
Working one (with okhttp)
```
D 2020-08-12 07:34:52:682 c:\projects\webdriver\cpp\webdriver-server\server.cc(338) Command: POST /session/f4bf25a3-8f7a-45fd-9939-e7744206b095/url {"url":"http://www.google.com"}
D 2020-08-12 07:34:52:682 c:\projects\webdriver\cpp\webdriver-server\command.cc(36) Raw JSON command: { "name" : "get", "locator" : { "sessionid" : "f4bf25a3-8f7a-45fd-9939-e7744206b095" }, "parameters" : {"url":"http://www.google.com"} }
```

**Comment by jimevans** [comment]
I believe this has been fixed in 2dfe468. To test this, you can get version 3.150.1.1 from the [Selenium source tree in the cpp/prebuilt directory](https://github.com/SeleniumHQ/selenium/tree/trunk/cpp/prebuilt/Win32/Release), and try that. Do note that at this time, that is the only way one can get this fix, aside from building from source. If that does not resolve the issue, please let me know.

**Comment by diemol** [comment]
I am going to close this as we did not get any reply after the previous comment, I assume it either worked or is not relevant anymore.
